Documentation/network/bgp-control-plane.rst

```diff
@@ -305,10 +305,15 @@ defaults to the ``LoadBalancerIP`` service. You can also specify the ``.serviceA
 field to advertise specific service types, with options such as ``LoadBalancerIP``,
 ``ClusterIP`` and ``ExternalIP``.
 
+It is worth noting that when you configure ``virtualRouters[*].serviceAdvertisements`` as ``ClusterIP``,
+the BGP Control Plane will only consider the configuration of the service's ``.spec.internalTrafficPolicy`` and ignore
+the configuration of ``.spec.externalTrafficPolicy``.
+For ``ExternalIP`` and ``LoadBalancerIP``,it will only consider the configuration of
+the service's  ``.spec.externalTrafficPolicy`` and ignore the configuration of ``.spec.internalTrafficPolicy``.
```

Comment:
Is it a change of the actual Control Plane's behaviour, and if so, should it be mentioned in the upgrade notes?

Reply:
I don't think it's necessary, this feature was just merged and there is already very clear context about it in the description of  #31389. 

This is just a supplement to the code and docs of PR #31331.  @qmonnet

Reply:
Ok thanks!